src/frontend/src/pages/GanttPage/GanttPageWrapper.tsx

```diff
@@ -20,50 +20,74 @@ import {
   sortTeamNames,
   GanttTask,
   transformProjectToGanttTask,
-  getProjectTeamsName
+  getProjectTeamsName,
+  EventChange
 } from '../../utils/gantt.utils';
 import { routes } from '../../utils/routes';
-import { useToast } from '../../hooks/toasts.hooks';
-import { Box, createTheme, Paper, ThemeProvider, Typography } from '@mui/material';
-import { nerThemeOptions, lightThemeOptions } from '../../utils/themes';
+import { Box, Popover, Typography, IconButton, useTheme, Chip, Tooltip, Card } from '@mui/material';
 import PageLayout from '../../components/PageLayout';
+import { GanttChartCalendar } from './GanttChartCalendar';
+import {
+  GanttWorkPackageStageColorPipe,
+  GanttWorkPackageTextColorPipe,
+  WbsElementStatusTextPipe,
+  WorkPackageStageTextPipe
+} from '../../utils/enum-pipes';
+import { SearchBar } from '../../components/SearchBar';
 
 /**
  * Documentation for the Gantt package: https://github.com/MaTeMaTuK/gantt-task-react
  */
 const GanttPageWrapper: FC = () => {
```

Comment:
This component is a beast, I would if possible break it out into a GanttChartPage component that wraps a component named GanttChart. Try keeping a lot of the logic/ query param stuff in GanttChartPage, and a lot of the React, component stuff in GanttChart. You could also probably break some of these components out into their own files

Reply:
I would make a folder called GanttChartComponents, under the GanttChartPage folder and put any components (e.g. the GanttChartElement component or GanttChartFilters) that don't contain any higher level organizational logic and are pretty isolated in their functionality/ placement on the screen.